src/components/tabController/TabBar.tsx

```diff
@@ -216,6 +217,7 @@ const TabBar = (props: Props) => {
           activeBackgroundColor={activeBackgroundColor}
           {...item}
           {...context}
+          style={[{flex}, item?.style]}
```

Comment:
Can you memoize this style, it might affect performance if you are passing inline array style